### ISSUE TITLE
Provide test types documentation

### ISSUE THREAD

**Issue by senpl** [Manthan933/Manthan#236]
**Is your feature request related to a problem? Please describe.**
Currently type is used in tests, but no description or anything describes what is it and how it differs

**Describe the solution you'd like**
Tooltip over type field to describe what this field is used for.

**Describe alternatives you've considered**
Wiki page with types description would also be nice, still I prefer tooltip. Also desciption during test creation would work, but this would change view a bit.

**Additional context**
Types are required during tests creation still they are not explained.
If more info is needed let me know.

**Comment by aavishkarmishra** [comment]
@senpl would you like to work on this issue ?

**Comment by senpl** [comment]
> @senpl would you like to work on this issue ?
No. I have no idea what types even mean in this project. But providing description would probably help at least in some understanding.

**Comment by aavishkarmishra** [comment]
> @aavishkarmishra can u please elaborate on the fact what test types actually mean here and what exactly do I need to do ??
Types refers to the type of a specific question in a test.
For example : 
Type 1 questions are of 1 marks,
Type 2 questions are of 2 marks,
Type 3 questions are of 3 marks,
Type 4 questions are of 4 marks,

By this way user can set marks for each type question and set the no of questions to be selected in tests.
